nemo/core/config/schedulers.py

```diff
@@ -72,13 +86,14 @@ class SquareRootAnnealingParams(WarmupSchedulerParams):
 
 
 @dataclass
-class CosineAnnealingParams(WarmupSchedulerParams):
+class CosineAnnealingParams(WarmupAnnealingHoldSchedulerParams):
     """
     Cosine Annealing parameter config
     It is not derived from Config as it is not a NeMo object (and in particular it doesn't need a name).
     """
 
     min_lr: float = 0.0
+    max_lr: float = 2.5e-4
```

Comment:
Revert. This should be set by the yaml config, not the default of the data class itself since it's domain dependent